SQL/sql.txt

```diff
@@ -0,0 +1,43 @@
+--CREATE TABLE: Creates an Employees table with columns for EmployeeID, FirstName, LastName, Department, and Salary.
+create table employees(emp_id int primary key,first_name varchar(30),last_name varchar(30),department varchar(30),salary int);
+
+--INSERT: Adds three employee records to the table.
+insert into employees values(1,'a','abin','cse',20000);
+insert into employees values(2,'b','benny','ce',15000);
+insert into employees values (3,'c','cyril','cse',13000);
+
+--Select all columns and rows
+select * from employees;
+
+--Select specific columns
+--Select with a WHERE clause to filter results
+select emp_id from employees where emp_id=1;
+select * from employees where emp_id=1;
+
+--Write a SELECT statement to retrieve only the FirstName and Salary of all employees.
+select first_name,last_name,salary from employees;
+
+--How would you modify the existing UPDATE statement to give all employees in the IT department a 10% raise
+update employees set salary=0.1*salary +salary from employees where department='cse';
+
+--Write a SELECT statement to find the highest salary in the Employees table.How would you add a new column named "HireDate" of type DATE to the Employees table?
+select max(salary) as maximum_salary from employees;
+alter table employees add HireDate date;
+
+--Write an INSERT statement to add a new employee named "Sarah Brown" in the "Marketing" department with a salary of 72000.00.
+insert into employees values(4,'Sarah','Brown','marketing',72000,NULL);
```

Comment:
Why this null value insertion ?

Reply:
What will happen if I do not mention the null value, Will it successfully insert the record?